vars/packages/cloud-init.suse.yml

```diff
@@ -23,7 +23,6 @@
 #
 ---
 packages:
-  - cloud-init-config-suse=23.3.3-1
-  - cloud-init=23.3.3-1
+  - cloud-init-config-suse=23.3-150100.8.79.2
+  - cloud-init=23.3-150100.8.79.2
   - supportutils-plugin-cloud-init=1.1-150400.1.5
```

Comment:
Hm, looks like we accidentally included our fork before for the cloud-init packages. Interesting.